keyform/views.py

```diff
@@ -3,16 +3,31 @@
 
 from django.views.generic import CreateView
 from django.views.generic.list import ListView
-from keyform.forms import CreateForm
+from keyform.forms import CreateForm, RequestFormSet
 from keyform.models import Request
 from django.urls import reverse_lazy
+from django.shortcuts import render
+from django.http import HttpResponseRedirect
+from django.core.urlresolvers import reverse
 
 class HomeView(ListView):
     model = Request
     template_name = "keyform/home.html"
 
-class NewForm(CreateView):
-    template_name = "keyform/add_form.html"
-    model = Request
-    success_url = reverse_lazy('home')
-    form_class = CreateForm
+def key_request(request):
```

Comment:
You're going to hate me for this: Make this into a class-based view that inherits from TemplateView